x-pack/plugins/security_solution/server/lib/detection_engine/signals/utils.ts

```diff
@@ -10,7 +10,7 @@ import moment from 'moment';
 import uuidv5 from 'uuid/v5';
 import dateMath from '@elastic/datemath';
 import type { estypes } from '@elastic/elasticsearch';
-import { isEmpty, partition } from 'lodash';
+import { chunk, isEmpty, partition } from 'lodash';
```

Comment:
It's ok if you want to use lodash, although still most people pull from `lodash/fp` but I don't see a big issue unless I see someone using one the mutatious things from lodash.